### DISCUSSION TITLE
How to resize TopNavigation Identity display

### DISCUSSION THREAD

**Discussion by suvihaara** [cloudscape-design/components#606] [Q&A] [answered]
Hi, 

I had an issue where when an SVG is scaled in % rather than a value, the TopNavigation title is truncated. I fixed it by changing the scaling of the SVG to be set in values rather than %. Is there a way that would allows the TopNavigation Logo SVGs to be scaled in % without truncating the title?

**Comment by timogasda** [comment]
Hi @suvihaara ! I'm not sure I completely understand. Could you please provide an example SVG with % and with values? And some screenshots of how those looked for you?

**Comment by suvihaara** [comment]
Hi @timogasda! No problem, hope this message will clarify the issue.

The original SVG file I downloaded to use as the logo file for the TopNavigation component had the following format:
![original_svg_format](https://user-images.githubusercontent.com/116554358/211278612-c4f3906d-b182-44d4-afa9-b00550bb79cf.png) SVG format checks did not raise any issues about the SVG height and width being set as percentages on line 10. This format resulted in the title of the web-page to truncate:
![failing_topnav](https://user-images.githubusercontent.com/116554358/211278931-8b07620c-adb1-40aa-ae6f-baba4f08d99b.png)

I spent quite a while trying to figure out why the title of the page was truncated in this manner. When I changed the width and height of the SVG on line 10 into this format:
![fixed_svg_format](https://user-images.githubusercontent.com/116554358/211279663-f3afceb2-9552-48bb-b89f-b33b82b2e793.png)
The logo and title displayed correctly:
![working_topnav](https://user-images.githubusercontent.com/116554358/211279836-f5232758-3a27-429c-b7cd-b23d4c4f20f2.png)

I just wanted to raise awareness of this in case there is a way to prevent the same issue from happening again. It was an easy thing to fix once I happened to stumble on the cause of it, but the cause was a bit of a random bug. 

Hope this explains the issue I had a bit better.

**Comment by Domt301** [comment]
Ran into the same issue and used this `    useEffect(() => {
        const span = document.getElementsByClassName("awsui_title_k5dlb_1a945_195")[0]
        span.style.overflow = "visible"
    }, []);`
like a caveperson lmao

**Comment by YueyingLu** [comment, marked as answer]
Converted this into an issue https://github.com/cloudscape-design/components/issues/704. Let's move the conversation here.

**Comment by taheramr** [comment]
Copying my reply in the [opened issue](https://github.com/cloudscape-design/components/issues/704) here for visibility:

Sorry for the late replies on this bug, this [PR](https://github.com/cloudscape-design/components/pull/826) should fix the issue with the top navigation and full width/height svg logos.

Will followup when this is merged.
@suvihaara @Domt301
